libs/langgraph/langgraph/pregel/executor.py

```diff
@@ -63,10 +63,14 @@ def submit(  # type: ignore[valid-type]
         __next_tick__: bool = False,
         **kwargs: P.kwargs,
     ) -> concurrent.futures.Future[T]:
+        ctx = copy_context()
         if __next_tick__:
-            task = self.executor.submit(next_tick, fn, *args, **kwargs)
+            task = cast(
+                concurrent.futures.Future[T],
+                self.executor.submit(next_tick, ctx.run, fn, *args, **kwargs),  # type: ignore[arg-type]
```

Comment:
nit: `next_tick` is a bit confusing on the sync side -- sleep(0) isn't guaranteed  to context switch

Reply:
It is the closest equivalent. And yes it does context switch https://stackoverflow.com/a/7273727

Reply:
Also not something being changed in this PR?